tests/apps/actorapp/.cache-include

```diff
@@ -0,0 +1 @@
+../utils/*.go
```

Comment:
at least for me, sounds more natural if the path is relative from root rather than relative from app directory,

but I don't have a strong opinion though

Reply:
I don't have a strong opinion either.

Reply:
Although thinking about it, implementation-wise, "root" is a bit harder to compute, because the build-tools app doesn't know what is the project's root. Doing that is possible but would require some other refactorings.

Reply:
I can live with that